airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -138,6 +138,8 @@ private boolean runProcess(final String jobId,
           attempt,
           jobRoot,
           normalizationImageName,
+          // custom connector does not use normalization
```

Comment:
@davinchia Can you help me confirm if this is true? (seems NormalizationRunner will only be running if it's called out here: https://github.com/airbytehq/airbyte/blob/master/airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java#L18-L38)

Reply:
I believe this is true, normalization is supported in a specific list of destination connectors, so I don't think we would be able to support normalization for custom destination connectors (at least for V1)

Reply:
Parker is correct!

Reply:
Perfect, thanks!